### PR TITLE
Notify if there is a very high rate of change in summaries

### PR DESCRIPTION
**Pull request by drewsamnick**
Adding a check to alert us if there is an unexpectedly high rate of change on census summaries. I set this to alert if more than 10% of the total schools change in any one categorization (yes, no, maybe, etc.) I'm open to alternative checks if you have any suggestions.

### CODE REVIEW

File: bin/cron/update_census_map [hunk 1]
```diff
@@ -25,7 +25,23 @@ def make_location(lat, long)
 end
 
 # Update summaries before updating the fusion table
+# If any single teaches_cs category in a given year changes by more than
+# 10% of the total number of schools then issue a Honeybadger nofication.
```
Comment (breville) [inline bin/cron/update_census_map]: typo: notification

File: bin/cron/update_census_map [hunk 2]
```diff
@@ -25,7 +25,23 @@ def make_location(lat, long)
...
+threshhold = School.count * 0.1
```
Comment (breville) [inline bin/cron/update_census_map]: typo: threshold

Comment (drewsamnick) [reply bin/cron/update_census_map]: fixed

**Comment by drewsamnick** [comment]
[Alert if a lot of schools change](https://trello.com/c/yYR75Ssm/181-alert-if-a-lot-of-schools-change)

**Pull request merged**